### PR TITLE
Develop landing page footer upd

### PR DESCRIPTION
**Pull request by nhefner12**
Modified the landing page footer to resemble Alex's wireframe on invision

### CODE REVIEW

**Comment by nhefner12** [comment]
need to add mobile support

**Comment by nhefner12** [comment]
added mobile support

**Comment by CSchmitz81** [comment]
For #120

File: src/ctree-app/ctree-landing.html [hunk 1]
```diff
@@ -290,34 +291,30 @@
         }
         /* ctree blub text formatting */
         #ctree-blurb p {
-          tex-align: center;
+          text-align: center;
```
Comment (CSchmitz81) [inline src/ctree-app/ctree-landing.html]: Good catch!

File: src/ctree-app/ctree-landing.html [hunk 2]
```diff
@@ -360,8 +357,8 @@
         padding: 36px 0 64px 0;
       }
       #footer.section-styling {
-        background-color: #2B2B2B;
-        color: #B4B4B4;
+        background-color: #FFFFFF;``
```
Comment (CSchmitz81) [inline src/ctree-app/ctree-landing.html]: Looks like there's a trailing ` `` `

File: src/ctree-app/ctree-landing.html [hunk 3]
```diff
@@ -511,45 +508,39 @@ <h1 align="left">Participate</h1>
...
+                <div class="flex-grid">
+                    <div class="flex-row">
+                      <div class="flex-column">
+                        <div id="ctree-blurb">
+                          <p>The Collaboration Tree is a project of the Foundation For an Innovative Future</p>
+                        </div>
+                      </div>
+                      <div class="flex-column">
+                        <div id="copyright">
+                          <p>&copy Foundation For an Innovative Future.</p>
```
Comment (CSchmitz81) [inline src/ctree-app/ctree-landing.html]: I never noticed this part of the comp.  It's actually an `@` not a copyright symbol, though I can see why you'd think that.  I can double check with Alex but the idea was probably to indicate it's a project of the larger nonprofit, maybe with a link to the nonprofit website (innovativeFuture.org).

**Comment by CSchmitz81** [comment]
OK, just tested this and it needs a bit of cleanup work.  With my background in front end mobile development I get pretty picky about matching comps, and it's an important skill to develop.  I recommend opening landing page in a window and the comp in a new tab of the same window, then size the window to the width of the comp and compare the tabs.  Here's what I spotted:
- the footer isn't as tall as the comp
- the "project of" text needs more margin on the left side (I think it's the same padding on the left/right sides of the window)
- the spacing between the sharing icons and the "project of" text needs to be increased slightly
- the space between the sharing icons needs to be increased slightly
- all text is italicized but shouldn't be
- the copyright text should be larger
- for mobile, the text has too much padding (it's much less for the mobile comp)
- for mobile, the text appears to be larger than the comp

On the issue of the copyright text, I actually agree with you using copyright instead of `@` (it makes a lot more sense), so go ahead and keep that difference.

**Comment by CSchmitz81** [comment]
Is this something you plan to work on during the hackathon this Saturday?

**Comment by nhefner12** [comment]
Yes, this was something I was planning on doing on or before Saturday.

**Comment by nhefner12** [comment]
Fixed up some of the comp dissimilarities.
Please review and let me know what you think.

**Pull request merged**